cli/cmd/initnodes.go

```diff
@@ -398,3 +418,35 @@ func gethInit(ctx context.Context, cfg InitConfig, dir string) error {
 
 	return nil
 }
+
+func downloadAndRestoreLatestSnapshot(ctx context.Context, network string, clientName string) error {
+	snapshotArchive := getSnapshotBackupArchive(clientName)
+	bucketName := fmt.Sprintf("omni-%s-snapshots", network)
+	gcpCloudStorageURL := fmt.Sprintf("https://storage.googleapis.com/%s/%s", bucketName, snapshotArchive)
```

Comment:
use less verbose variables, e.g. `url`

Reply:
can this be collapsed into single inline?
```
url := fmt.Sprintf("https://storage.googleapis.com/omni-%s-snapshots/%s_data.tar.lz4", network, client)
```

Reply:
Done.